src/javascript/app_2/Modules/Trading/Components/Form/ContractType/contract_type_list.jsx

```diff
@@ -11,6 +11,7 @@ const ContractTypeList = ({
 }) =>
     (
         Object.keys(list).map(key => (
+            !['In/Out', 'Asians'].includes(key) &&
```

Comment:
can you add a todo comment here to remove this when we want to enable other trade types?